screwdriver/screwdriver.go

```diff
@@ -3,13 +3,20 @@ package screwdriver
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
 	"net/http"
 	"net/url"
+	"time"
+
+	"github.com/jpillora/backoff"
```

Comment:
I don't want to use a library for this. It's too easy to write a retry wrapper and not worth pulling a dependency in.